spring-boot-project/spring-boot-docs/src/docs/asciidoc/messaging/pulsar.adoc

```diff
@@ -7,7 +7,9 @@ It will do the same for the reactive components when `org.springframework.pulsar
 
 There are `spring-boot-starter-pulsar` and `spring-boot-starter-pulsar-reactive` "`Starters`" for conveniently collecting the dependencies for imperative and reactive use, respectively.
 
-
+TIP: The Reactive support is built on top of the imperative support and therefore the reactive starter includes the imperative starter.
```

Comment:
I thought it may be helpful to list this fact and tell them how to "hide the non-reactive" components.

Reply:
I removed this initially because the reactive starter doesn't include the imperative starter (although the reactive starter does include imperative dependencies). I've removed it again for now as I think this kind of detail probably belongs in the Spring Pulsar docs rather than ours.